pre-docs/bolt_configuration_options.md

```diff
@@ -225,3 +225,25 @@ The `puppetfile` section configures how modules are retrieved when running `bolt
 `proxy`: The HTTP proxy to use for Git and Puppet Forge operations
 
 `forge`: A subsection which can have its own `proxy` setting to set an HTTP proxy for only Puppet Forge operations, and a `baseurl` setting to specify a different Forge host
+
+## Plugin hooks configuration options
+
+The `plugin_hooks` section allows you configure what plugins are used for a hook on a per target basis. This section is a hash where the keys are hooknames and the value specifies and configures the plugin to use. Currently there are two plugins you'll want to use for this: your own custom task using the `task` plugin, or the `install_agent` plugin which runs the `puppet_agent::install` task.
+
+For now, the only configurable plugin hook is the `puppet_library` plugin hook.
+
+```
+plugin_hooks:
+  puppet_library:
+    plugin: install_agent
+```
+
+```
+plugin_hooks:
```

Comment:
We usually prefer hyphens in the inventory file. That may be a bit confusing since the plugin names are `_`